src/lib/y2journal/entry_presenter.rb

```diff
@@ -52,8 +52,14 @@ def formatted_time
 
     # Message string
     def message
+      msg = entry.message
       # bnc#941655 was caused by this field being an array in journalctl's json
-      entry.message.to_s
+      # Also big arrays are problematic
+      if msg.is_a?(Array)
+        "[#{msg.size} bytes blob data]"
```

Comment:
What about i18n?

Reply:
So we need to add `include Yast::i18n` to the class, add `textdomain "journal"` to the constructor and use the i18n placeholders our translators are used to. Something like:

```ruby
_("Blob data (%i bytes)") % msg.size
```

I changed the text a bit since it makes the sentence more understandable to me, but feel free to leave it as it was.